### ISSUE TITLE
Simplify input

### ISSUE THREAD

**Issue by gbishop** [benknoble/limchess#25]
It's a bother having to tab all around the interface when I mostly want next move and select move. I suggest you define two more keys (maybe left and right arrow) bound to those functions.

**Comment by benknoble** [comment]
Good thought; I didn’t notice that much when I used shift-tab, but I suppose that’s a hard combination given motor-impairments. Thanks for the advice.

**Comment by gbishop** [comment]
Your typical user has ONLY 2 buttons. It's a huge pain. It takes a lot of
thought to optimize the number of key hits required.

**Comment by zlintz14** [comment]
Fixed! You can now use the left and right arrow keys on any page to navigate the same items you would've with tab and enter. Still use enter to click/select items. For sliders, hit enter, then use the left/right arrow keys to adjust, then hit enter again to confirm your choice and go back to normal navigation mode. Help page has been updated. Will update description of YouTube help video tomorrow to reflect changes.

**Comment by benknoble** [comment]
@zlintz14 I think @gbishop’s point was to have left switch moves and right play the move; can you confirm Professor?

**Comment by gbishop** [comment]
Actually it's right switch moves and left switch selects. Look at Tar Heel
Reader.

None of this is required, I'm just trying to help you get the best result.

**Comment by zlintz14** [comment]
Ah whoops! I fixed this now! You now use the right arrow key to navigate the site, and the left arrow key to select, like the Tar Heel Reader. If you are over a slider, use the left arrow key to select, then use the right arrow key to adjust (if you get to the max of the slider and press right, it resets to the min value). Press the left arrow key again when you are done adjusting the slider to confirm your choice. All this has been updated on the help page. I'll update the Youtube video later today.
